integration-tests/wallet-no-annotations-call-by-index.spec.ts

```diff
@@ -3,13 +3,13 @@ import { noAnnotCode, noAnnotInit } from "./data/token_without_annotation";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test contract made with wallet API with no annotations calling methods by index using: ${rpc}`, () => {
+  describe(`Test contract origination with no annotations calling methods by index through wallet api using: ${rpc}`, () => {
```

Comment:
I think the intention of the test is not the origination of the contract but calling its methods using indexes.
Of course, we do an origination because we need the contract to interact with.

Is it worth mentioning "contract origination" in the description?

Reply:
agree

changed to
"Test calling methods by index through wallet api using: ${rpc}`"